locale/en/docs/guides/buffer-constructor-deprecation.md

```diff
@@ -31,10 +31,11 @@ If you’re using Node.js ≥ 8.0.0 (which is recommended), Node.js exposes mult
 - `--trace-deprecation` does the same thing, but only for deprecation warnings.
 - `--pending-deprecation` will show more types of deprecation warnings. In particular, it will show the `Buffer()` deprecation warning, even on Node.js 8.
 
-You can set these flags using an environment variable:
+You can set these flags using environment variables:
 
-```console
-$ export NODE_OPTIONS='--trace-warnings --pending-deprecation'
+```bash
```

Comment:
`console` tag seems to produce [weird rendering](https://nodejs.org/en/docs/guides/buffer-constructor-deprecation/#finding-problematic-bits-of-code-using-node-js-8):

![1](https://user-images.githubusercontent.com/10393198/40885599-1faf871c-6732-11e8-9447-91a7b15e457b.png)